pkg-fun.nix

```diff
@@ -38,12 +40,16 @@
   };
   propagatedBuildInputs = [semver nix.dev boost];
   nativeBuildInputs     = [pkg-config];
-  buildInputs           = [sqlite.dev nlohmann_json argparse sqlite3pp];
-  nix_INCDIR            = nix.dev.outPath + "/include";
-  boost_CFLAGS          = "-I" + boost.dev.outPath + "/include";
-  libExt                = stdenv.hostPlatform.extensions.sharedLibrary;
-  SEMVER_PATH           = semver.outPath + "/bin/semver";
-  configurePhase        = ''
+  buildInputs           = [
+    sqlite.dev nlohmann_json argparse sqlite3pp toml11 yaml-cpp
+  ];
+  nix_INCDIR     = nix.dev.outPath + "/include";
```

Comment:
You're just going to sit there and let the `=` be on a different column than the previous line, like an animal?

Reply:
wait did I botch the alignment?

Reply:
No my alignment is correct.
Line 37 lacks an `=`, resetting the column.
I'm going to switch those lines though to make them more pretty.

Also in case you didn't know, there's a gear icon at the top of the GitHub UI that allows you to ignore whitespace changes.
Because I frequently play with alignment I recommend using it.